docs/feature-engineering.rst

```diff
@@ -37,6 +37,19 @@ The statistics available include:
 
 Putting the operation and the statistic together, you can see some examples. To get stddev of classic_idf, you would write :code:`stddev_classic_idf`. To get the minimum total term frequency, you'd write :code:`min_raw_ttf`.
 
+Term position statistics
+
+The :code:`type` parameter can be prepended with the operation to be performed across term position for the statistic :code:`max`, :code:`min`, :code:`avg`, and :code:`norma_avg`. The :code:`norma_avg` is a normalization average using a specific pound when second position does not exists in document.
```

Comment:
May need to clean this phrasing up a little bit.  I'm confused by "normalization average using a specific pound when second position does not exists in document.", also some small cleanup needed below.